gamePlay.js

```diff
@@ -0,0 +1,73 @@
+import { displayRevealedCell, revealWholeGrid, resetDisplay} from "./display.js";
+
+
+export const changeDisplayedCell = function (cell, row, col, gridObject, game) {
+    let currentCell = gridObject.getCell(row, col);
+    console.log(gridObject.getCell(row, col));
+    if (game.gameOver){
+        return;
+    }
+
+    if (game.flaggingOn){
+        let cellObject = gridObject.getCell(row, col);
+        toggleFlagging(cellObject, cell);
+        return;
+    }
+
+    if (currentCell.numMinesAround == 0){
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        const revealedCells = gridObject.revealZeros(row, col);
+        revealedCells.forEach(c =>{
+            displayRevealedCell(c);
+        } );
+        return;
+    };
+    if (currentCell.isMine){
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        revealWholeGrid(gridObject);
+        game.gameOver = true;
+        return;
+    }
+    else {
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        displayRevealedCell(currentCell);
+        return;
+    }
+    
+};
+
+export const clickToReveal = function (gridObject, game){
+    for (let row = 0; row < gridObject.gridSize; row++) {
+        for (let col = 0; col < gridObject.gridSize; col++) {
+            let cell = document.querySelector(".row" + row + "col" + col);
```

Comment:
prefer string formatting over concatenation (i.e. using `+`), I'd also suggest separating with `-` for readability. 
this is also done multiple times throughout your codebase, I would extract it to it's own function
```suggestion
            let cell = document.querySelector(`.row-${row}-col-${col}`);
```

Reply:
done